.ci.yaml

```diff
@@ -31,6 +31,8 @@ targets:
     recipe: cocoon/device_doctor
     properties:
       add_recipes_cq: "true"
+      script: "build.sh"
```

Comment:
nit: you don't need the quotation marks, yaml should convert this to a string

Reply:
sure, updated